Statically compile FEX?

Is there a way to statically compile FEX, so it does not need the presence of the aarch64 loader, similar to QEMU? I tried `CC=clang CXX=clang++ LD=ld cmake -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_BUILD_TYPE=Release -DENABLE_LTO=False -DENABLE_STATIC_PIE=True -DBUILD_TESTS=False -DENABLE_ASSERTIONS=False -G Ninja ..`, but it failed with
```
CMake Error at CMakeLists.txt:211 (message):
  Couldn't compile static-pie test.  Static-pie can't be enabled!
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -208,7 +208,7 @@ if (ENABLE_STATIC_PIE)
       message (FATAL_ERROR "Application has __rela_iplt_{start,end} symbols. Which means static-pie can't be enabled")
     endif()
   else()
-    message (FATAL_ERROR "Couldn't compile static-pie test. Static-pie can't be enabled!")
+    message (FATAL_ERROR "Couldn't compile static-pie test. Static-pie can't be enabled! Is your glibc compiled without static-pie?")
   endif()
 endif()
 

